lib/firebase_pagination.dart

```diff
@@ -12,3 +12,4 @@ export 'src/firestore_pagination.dart';
 // Data Models
 export 'src/models/view_type.dart';
 export 'src/models/wrap_options.dart';
+export 'src/models/filter_model.dart';
```

Comment:
Move this import above `export 'src/models/view_type.dart';` to follow alphabetical order